contracts/packages/math/float.fc

```diff
@@ -80,3 +80,16 @@ int safeDiv(int floatA, int floatB) inline_ref {
     return (floatA << 64) / floatB;
```

Comment:
IDK if `floatA << 64` can cause overflow with a big float

Reply:
Division will never cause overflow. Since floatA is always a 196 bit integer, so when it is shifted left by 64 bit, it will be 256 bit integer, still safe.

Reply:
I see. btw I think 196 is a typo (should be 192)